Makefile

```diff
@@ -16,7 +16,8 @@ src/epgsql_errcodes.erl:
 	./generate_errcodes_src.sh > src/epgsql_errcodes.erl
 
 test: compile
-	@$(REBAR) do ct -v
+	@$(REBAR) do ct -v -c
+	@$(REBAR) cover -v -m 55
```

Comment:
Could use a comment or maybe some notes somewhere explaining what this is: it's not obvious from a glance.

Reply:
Good idea! Will do.

Reply:
Made it a named constant. Also added eunit because we actualy have 2 unit-tests

Reply:
Or you mean to explain what `cover` is?

Reply:
Something like

# Fail the build if coverage falls below 55%